frontend/vite.config.ts

```diff
@@ -61,5 +61,10 @@ export default defineConfig(() => {
       plugins: () => [tsconfigPaths()],
       format: 'es' as const,
     },
+    define: {
+      // On local dev, global is undefined, causing decryption workers to fail.
+      // This is a workaround based on https://github.com/vitejs/vite/discussions/5912.
+      global: 'globalThis',
```

Comment:
used the bypass an error faced as described by the comment.

Reply:
I experimented with removing the whole chunk in [decryption.worker.ts](https://github.com/opengovsg/FormSG/blob/feat/hawkeye-v1.0/frontend/src/features/admin-form/responses/ResponsesPage/storage/worker/decryption.worker.ts#L18) and it seemed to have resolved the csv generation issue on local.

Might want to look into that solution since I would prefer deleting code than to have more config.

Reply:
removed the global reference + vite config, tested and it works well!